Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```diff
@@ -151,6 +151,12 @@
   - type: MobPrice
     price: 1000 # Living critters are valuable in space.
   - type: Perishable
+  - type: AmbientSound #Ideally in the future this will support multiple sources
+    enabled: false
+    volume: -5
+    range: 3
+    sound:
+      path: /Audio/Ambience/Temporary/flies.ogg
```

Comment:
yeah this is definitely sus right now. making the flies a separate entity is probably the best solution overall but need another opinion on that

Reply:
Mmmm, would an entity that just holds the ambient sound stuff that gets parented to entities with FliesComponent and removed when it's shut down be decent?